www/src/components/package-readme.js

```diff
@@ -45,35 +49,61 @@ const PackageReadMe = props => {
               justifyContent: `space-between`,
             }}
           >
-            {githubUrl.indexOf(`https://github.com/gatsbyjs/gatsby`) === 0 &&
-              packageName[0] !== `@` && (
-                <div
+            {isOfficial ? (
+              <div
+                sx={{
+                  variant: `links.muted`,
+                  mr: 8,
+                  "&&": {
+                    border: 0,
+                    color: `textMuted`,
+                    display: `flex`,
+                    fontWeight: `body`,
+                  },
+                  "&&:hover": {
+                    color: `textMuted`,
+                  },
+                }}
+              >
+                <span
                   sx={{
-                    variant: `links.muted`,
-                    mr: 8,
-                    "&&": {
-                      border: 0,
-                      color: `textMuted`,
-                      display: `flex`,
-                      fontWeight: `body`,
-                    },
-                    "&&:hover": {
-                      color: `textMuted`,
-                    },
+                    display: `inline-block`,
+                    mr: 2,
                   }}
+                  title={`Official Gatsby Plugin`}
                 >
-                  <span
-                    sx={{
-                      display: `inline-block`,
-                      mr: 2,
-                    }}
-                    title={`Official Gatsby Plugin`}
-                  >
-                    <GatsbyIcon />
-                  </span>
-                  Official Plugin
-                </div>
-              )}
+                  <GatsbyIcon />
+                </span>
+                Official Plugin
+              </div>
+            ) : (
+              <div
+                sx={{
+                  variant: `links.muted`,
+                  mr: 8,
+                  "&&": {
+                    border: 0,
+                    color: `textMuted`,
+                    display: `flex`,
+                    fontWeight: `body`,
+                  },
+                  "&&:hover": {
+                    color: `textMuted`,
+                  },
+                }}
+              >
+                <span
+                  sx={{
+                    display: `inline-block`,
+                    mr: 2,
+                  }}
+                  title={`Community Gatsby Plugin`}
+                >
+                  <CommunityIcon />
+                </span>
+                Community Plugin
+              </div>
+            )}
```

Comment:
Thank you for adding this icon 👍 

Can we abstract the component out of the page? Something along these lines? It makes it easier to read.

```js
const GatsbyPluginBadge = ({ isOfficial }) => {
  const Icon = isOfficial ? GatsbyIcon : CommunityIcon;
  const title = isOfficial
    ? "Official Gatsby Plugin"
    : "Community Gatsby Plugin";
  const text = isOfficial ? `Official Plugin` : `Community Plugin`;

  return (
    <div
      sx={{
        variant: `links.muted`,
        mr: 8,
        "&&": {
          border: 0,
          color: `textMuted`,
          display: `flex`,
          fontWeight: `body`
        },
        "&&:hover": {
          color: `textMuted`
        }
      }}
    >
      <span
        sx={{
          display: `inline-block`,
          mr: 2
        }}
        title={title}
      >
        <Icon />
      </span>
      {text}
    </div>
  );
};

```

```suggestion
              <GatsbyPluginBadge isOfficial={isOfficial} />
```

Reply:
> Thank you for adding this icon 👍

Looks like we might upgrade `react-icons` to 3.9. We will have this icon in that version. https://github.com/gatsbyjs/gatsby/pull/21820#issuecomment-592797020

> Can we abstract the component out of the page? Something along these lines? It makes it easier to read.

Sure. I'll do that.

Reply:
Just an inline component is fine :) Just put it at the top of this file.

> Looks like we might upgrade react-icons to 3.9. We will have this icon in that version. #21820 (comment)

Awesome!